Dockerfile

```
@@ -36,7 +36,7 @@ RUN pip install --quiet --upgrade pip setuptools wheel &&  \
 # Need to use specific cuda versions for jax
 ARG USE_CUDA=true
 RUN if [ "$USE_CUDA" = true ] ; \
-    then pip install "jax[cuda11_pip]<=0.4.13" -f "https://storage.googleapis.com/jax-releases/jax_cuda_releases.html" ; \
+    then pip install "jax[cuda12]==0.4.25" ; \
```

Is the ` pip install "jax[cuda12]==0.4.25"` enough?
```suggestion
    then pip install "jax[cuda12_pip]==0.4.25" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html"; \
```

---

Yup it should be, I've tested locally and it works fine